modules/i18n/src/translations/en-US/portals/organizations.ts

```diff
@@ -137,7 +137,7 @@ export const organizations: organizationsNS = {
             orgHandle: {
                 label: "Organization Handle",
                 placeholder: "Organization handle (E.g., myorg.com)",
-                tooltip: "A human-readable unique identifier used as a URL handle to access the new organization.",
+                tooltip: "A human-readable unique identifier for handle-based organization discovery in your B2B applications. Leave this field blank to use the organization ID as the handle.",
```

Comment:
```suggestion
                tooltip: "A human-readable unique identifier for organization discovery in your B2B applications. Leave this field blank to use the organization ID as the handle.",
```

Handle based is implied in this case and need not to be explicitly mentioned imo

Reply:
This is now updated.